vendor/jquery-ui/AUTHORS.txt

```diff
@@ -0,0 +1,333 @@
+Authors ordered by first contribution
```

Comment:
We shouldn't vendorize jquery-ui or other dependencies like that. We can install them via npm/bower and import them from there if needed.

Reply:
@pzuraq I don't know how to import jq-ui into the app. The methods provided are chained to jq.